index.js

```diff
@@ -115,6 +115,11 @@ function vine(str) {
  * @returns {string|undefined}
  */
 function youtube(str) {
+
+	// remove time and feature
+	str = str.replace(/#t=.*/, "");
```

Comment:
Maybe we should put in some protection here, because the greediness of this regex makes me nervous (where it will remove *everything* after the `#t=`) I think in general we should probably be safe to assume that the time parameter comes at the very end of the url, but for future proofing, and general saftey, maybe for now we should add an `$` at the end of the regex to denote the "end of the string" just to be sure that we are removing the time parameter from the end of the url 

```suggestion
	str = str.replace(/#t=.*$/, "");
```